core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/SamDAO.scala

```diff
@@ -76,4 +78,9 @@ object SamDAO {
   case object NotFound extends GetUserIdInfoResult
   case object NotUser extends GetUserIdInfoResult
   final case class User(userIdInfo: UserIdInfo) extends GetUserIdInfoResult
-}
\ No newline at end of file
+  val defaultScopes = Set(
+    "https://www.googleapis.com/auth/userinfo.email",
+    "https://www.googleapis.com/auth/userinfo.profile"
+  )
+  val bigQueryReadOnlyScope = "https://www.googleapis.com/auth/bigquery.readonly"
```

Comment:
```
  com.google.api.services.oauth2.Oauth2Scopes.USERINFO_EMAIL
  com.google.api.services.oauth2.Oauth2Scopes.USERINFO_PROFILE
  com.google.api.services.bigquery.BigqueryScopes.BIGQUERY_READONLY
```
should give you Google's definitions for these scopes, instead of hardcoding them